force-app/tests/Shared Code/OrgShape_Tests.cls

```diff
@@ -152,15 +151,15 @@ class OrgShape_Tests {
     @isTest(SeeAllData=true)
     @SuppressWarnings('PMD.ApexUnitTestShouldNotUseSeeAllDataTrue')
     static void testSeeAllDataPositiveWhenEnabled() {
-        System.assert(
+        System.Assert.isTrue(
             orgShape.isSeeAllDataTrue(),
             'Expected see all data to be true in a test method annotated with seeAllData=true'
         );
     }
 
     @isTest
     static void testSeeAllDataFalseWhenDisabled() {
-        System.assert(
+        System.Assert.isTrue(
```

Comment:
Change this to `isFalse` and drop the `!`

Reply:
I don't agree with this special case. Considering the business case that's asserted here, we want to assert with the `isTrue` since we are expecting true.

`orgShape.isSeeAllDataTrue()` is false by default, so we want to validate that it's changed to true by the code, so we negate and assert it's `true.`

Reply:
OK, fair.